instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/ConfigPropertiesUtil.java

```diff
@@ -17,11 +23,51 @@
  */
 public final class ConfigPropertiesUtil {
 
+  private static final boolean isIncubator = isIncubator();
+
+  private static boolean isIncubator() {
+    try {
+      Class.forName("io.opentelemetry.api.incubator.ExtendedOpenTelemetry");
+      return true;
+    } catch (ClassNotFoundException e) {
+      // The incubator module is not available.
+      // This only happens in OpenTelemetry API instrumentation tests, where an older version of
+      // OpenTelemetry API is used that does not have ExtendedOpenTelemetry.
+      // Having the incubator module without ExtendedOpenTelemetry class should still return false
+      // for those tests to avoid a ClassNotFoundException.
```

Comment:
> This only happens in OpenTelemetry API instrumentation tests

these are tests of OpenTelemetry API in the users class loader, so not following how that affects this class which is in the agent?